.gitignore

```diff
@@ -342,3 +342,23 @@ healthchecksdb
 #Sample Android Resource Designer.
 /Samples/Sample.Xamarin.Droid/Resources/Resource.designer.cs
 /Tests/Sentry.Xamarin.Forms.Tests.UWP/Sentry.Xamarin.Forms.Tests.UWP.GeneratedMSBuildEditorConfig.editorconfig
+
+
+# com.sentry.ovenlibrary Sample
+/Samples/Forms/com.sentry.ovenlibrary/AndroidProject/*.iml
```

Comment:
What about adding a `.gitignore` on that directory instead?

Also some stuff could go top level here like:

```
.DS_Store
```

https://github.com/getsentry/sentry-dotnet/blob/main/.gitignore#L6-L15